enterprise_subsidy/settings/base.py

```diff
@@ -297,3 +297,7 @@ def root(*path_fragments):
 
 # Set up logging for development use (logging to stdout)
 LOGGING = get_logger_config(debug=DEBUG)
+
+
+# Application settings
+ALLOW_LEDGER_MODIFICATION = False
```

Comment:
Just some forward-looking stuff to help with the Django admin experience around subsidy/ledger creation for test accounts.